File: abc/abc121/abc121_a.cs
// https://atcoder.jp/contests/abc121/submissions/9279368

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            var H = input[0];
            var W = input[1];

            input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
            var h = input[0];
            var w = input[1];

            Console.WriteLine($"{(H - h) * (W - w)}");
        }
    }
}

File: abc/abc121/abc121_a.fs
// https://atcoder.jp/contests/abc121/submissions/12277497

let [| H; W |] = stdin.ReadLine().Split() |> Array.map int
let [| h; w |] = stdin.ReadLine().Split() |> Array.map int

(H - h) * (W - w) |> stdout.WriteLine
